README.md

```diff
@@ -19,6 +20,33 @@ To install simply run:
 
   Be sure to not have a trailing `/` after `<YOUR SITE ROOT URL>` or else you will get double `//` in your generated **sitemap.xml**
   
+  Custom values for `changeFrequency` and `defaultPriorityValue`, as well as routes to `ignore` and `customPriority` values are optional.  Please add these to your environment.js file as shown in the example below
+  
+  ```js
+    <!-- environment.js -->
+    ...
+    
+    ENV['sitemap-autogenerator'] = {
+    changeFrequency: 'weekly', // Optional (if not included, default value is 'daily')
+    defaultPriorityValue: '0.3', // Optional (if not included, default value is '0.5')
+    showLog: true, // Optional (if not included, default value is false)
+    ignore: { // Optional (if not included, all routes will be included in sitemap.xml except those with path "*"
+      'contact-us': 'contact-us',
+      'contact': 'contact',
+      'algorithmictradedeveloper': 'algorithmictradedeveloper',
+      'careers': 'careers'
+    },
```

Comment:
Thanks for working on this!

In the course of testing this PR, I'm trying to follow your new instructions for using `ignore`.  The way I'm thinking about this, the end goal is to prevent certain paths from showing up in the 'sitemap.xml'.  Given that, I was expecting to provide an array of paths, and I'm not sure I understand what the keys and values are supposed to represent in the `ignore` object.  Can you please clarify this in the documentation?